op-acceptance-tests/tests/interop/proofs/challenger_test.go

```diff
@@ -26,7 +26,11 @@ func TestChallengerPlaysGame(gt *testing.T) {
 	attacker := sys.FunderL1.NewFundedEOA(eth.Ether(15))
 	dgf := sys.DisputeGameFactory()
 
-	game := dgf.StartSuperCannonGame(attacker, proofs.WithRootClaim(badClaim))
+	super, err := sys.Supervisor.FetchSuperRootAtTimestamp(sys.Supervisor.FetchSyncStatus().SafeTimestamp).ToSuper()
+	t.Require().NoError(err)
+	badSuper := super.(*eth.SuperV1)
+	badSuper.Chains[0].Output = [32]byte(badClaim)
+	game := dgf.StartSuperCannonGame(attacker, proofs.WithSuper(badSuper))
```

Comment:
oof, this is pretty ugly to have in the test itself. Can we change the DSL to manage this for us? Maybe even just `proofs.WithSuperRootFrom(...outputRoots)` and have the DSL fill in the timestamp.